### DISCUSSION TITLE
Сортировка по мета-значению при выводе элементов на странице

### DISCUSSION THREAD

**Discussion by W1SE95** [maxsite/cms#495] [Q&A] [answered]
Здравствуйте.

Есть такая кастомная страница: (type/custom-page/custom-page.php)
**Вывод в дней:**
if ($pages) {
foreach ($pages as $page) { 
$rating_admin_cryptoexchange = $page['page_meta']['rating_admin_cryptoexchange'][0];
и др. мета данные

}
}

Как по определенному мета (где указываются целые числа 90,80 и т.д. ... ) сортировать вывод элементов?
Спасибо!
![Screenshot_79](https://user-images.githubusercontent.com/60783381/179232637-9168e79f-6b32-4b30-b4de-2e259f1fd07a.jpg)

**Comment by maxsite** [comment, marked as answer]
Если` $pages` уже получен, то пропустить его через пользовательскую функцию сортировки, вроде `uasort()`.
